inc/patterns/gallery.php

```diff
@@ -8,5 +8,5 @@
="wp-block-themeisle-blocks-advanced-column"><!-- wp:gallery {"columns":2,"linkTo":"none","otterConditions":[],"isMasonry":true} --><figure class="wp-block-gallery has-nested-images columns-2 is-cropped"><!-- wp:image {"id":310,"sizeSlug":"large","linkDestination":"none"} --><figure class="wp-block-image size-large"><img src="https://api.themeisle.com/sites/images/cb:otter/q:auto/wp-content/uploads/sites/664/2022/09/otter-demo-02.png" alt="" class="wp-image-310"/></figure><!-- /wp:image --><!-- wp:image {"id":245,"sizeSlug":"large","linkDestination":"none"} --><figure class="wp-block-image size-large"><img src="https://mllj2j8xvfl0.i.optimole.com/w:300/h:300/q:mauto/rt:fill/g:ce/https://demosites.io/otter/wp-content/uploads/sites/664/2022/09/image-from-rawpixel-id-1054569-jpeg-1.png" alt="" class="wp-image-245"/></figure><!-- /wp:image --><!-- wp:image {"id":261,"sizeSlug":"large","linkDestination":"none"} --><figure class="wp-block-image size-large"><img src="https://demosites.io/otter/wp-content/uploads/sites/664/2022/09/image-from-rawpixel-id-1054551-jpeg-1024x715.jpg" alt="" class="wp-image-261"/></figure><!-- /wp:image --><!-- wp:image {"id":309,"sizeSlug":"large","linkDestination":"none"} --><figure class="wp-block-image size-large"><img src="https://api.themeisle.com/sites/images/cb:otter/q:auto/wp-content/uploads/sites/664/2022/09/otter-demo-03.png" alt="" class="wp-image-309"/></figure><!-- /wp:image --></figure><!-- /wp:gallery --></div><!-- /wp:themeisle-blocks/advanced-column --><!-- wp:themeisle-blocks/advanced-column {"id":"wp-block-themeisle-blocks-advanced-column-631489be","padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"paddingMobile":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","right":"12px","bottom":"0px","left":"12px"},"marginMobile":{"right":"0px","bottom":"24px","left":"0px"},"border":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"columnWidth":"30.00"} --><div id="wp-block-themeisle-blocks-advanced-column-631489be" class="wp-block-themeisle-blocks-advanced-column"><!-- wp:themeisle-blocks/advanced-heading {"id":"wp-block-themeisle-blocks-advanced-heading-1d40683c","tag":"p","fontSize":12,"fontSizeTablet":12,"fontSizeMobile":12,"textTransform":"uppercase"} --><p id="wp-block-themeisle-blocks-advanced-heading-1d40683c" class="wp-block-themeisle-blocks-advanced-heading wp-block-themeisle-blocks-advanced-heading-1d40683c"><strong>Gallery section</strong></p><!-- /wp:themeisle-blocks/advanced-heading --><!-- wp:themeisle-blocks/advanced-heading {"id":"wp-block-themeisle-blocks-advanced-heading-0af3bf17"} --><h2 id="wp-block-themeisle-blocks-advanced-heading-0af3bf17" class="wp-block-themeisle-blocks-advanced-heading wp-block-themeisle-blocks-advanced-heading-0af3bf17">A gallery with a text column and some text</h2><!-- /wp:themeisle-blocks/advanced-heading --></div><!-- /wp:themeisle-blocks/advanced-column --></div></div><!-- /wp:themeisle-blocks/advanced-columns -->',
```

Comment:
I see in some places the domain is still needs to be replaced with api.themeisle format.